frontend/src/App.js

```diff
@@ -453,7 +455,7 @@ const App = () => {
                <SplashPage />
           }/>
           <Route path='/' element ={
-           <LandingPage/>
+           <LandingPage externalIdentifier={user.email} groups={[5,6]} pageUuid={RETOOL_LANDING_PAGE_UUID} />
```

Comment:
note:
seems like this retool landing page uuid corresponds to the "Onboarding Page" app. looking at groups 5 and 6 (Gold and Silver), we haven't given those groups permissions to this app so you'll need to add for this to work.